src/Dofs/DofHandler.jl

```diff
@@ -127,6 +127,9 @@ add!(dh, :u, ip_u)
 add!(dh, :p, ip_p)
 close!(dh)
 ```
+
+!!! note
```

Comment:
```suggestion
!!! note "Dof numbering"
```
(Or DoF numbering, but we are a bit inconsistent here...)